include/GafferArnold/ArnoldDisplacement.h

```diff
@@ -81,6 +81,9 @@ class ArnoldDisplacement : public GafferScene::Shader
 
 		virtual void affects( const Gaffer::Plug *input, AffectedPlugsContainer &outputs ) const;
 
+		virtual void loadShader( const std::string &shaderName, bool keepExistingValues=false );
```

Comment:
Could you add a "Breaking Change" section to the commit message mentioning the addition of a virtual method, so I can make sure it gets into the release notes for the next major version?